### ISSUE TITLE
ü instead of u when convertng Devanagari to IASTpali

### ISSUE THREAD

**Issue by bdhrs** [virtualvinodh/aksharamukha#176]
i'm using the python module to convert Devanagiri to Roman `transliterate.process("autodetect", "IASTPali", text_extract)` and the resulting text has the letter `ü` instead of `u`. e.g. `saüddesaṃ` instead of `sauddesaṃ`. ü is non-existent in Pāli texts.

**Comment by virtualvinodh** [comment]
That's to differentiate between "sau" and "sa_u" (with a vowel hiatus).

Apparently, this is a often used convention in Prakrit texts, where vowel
hiatus is quite common.

V

**Comment by bdhrs** [comment]
pāli has a fixed system of 41 written letters. so which system do you recommend i use? at the moment i just do a regex sub afterwards.

**Comment by virtualvinodh** [comment]
The problem is romanization.

In Indic systems, /sau/ and /sa_u/ are graphically different and both
cannot be confused with each other. However, when you romanize them both
end up as /sau/, hence the need for disambiguation.

It also depends why you want to romanize them. If you think, the
reader/user wouldn't read /sauddesam/ as saud-de-sam but rather
sa-ud-de-sam (because Pali doesn't have /au/ and any /au/ sequence must be
read as /a-u/ with a hiatus), you can ignore the disambiguation.

Even if it's for some internal machine learning purpose, I would still
differentiate it, since any /au/ sequence most probably indicates a
morpheme boundary (sa-uddesam) in your example.

TLDR; it depends. If you know how Pali works and doesn't confuse it for
they vowel /au/ then just don't show the distinction. Else, you should
disambiguate it by some method (I have chosen the umlaut)

V

**Comment by bdhrs** [comment]
ok great i understand where you are coming from.

if you look at a large pāli corpus like chaṭṭha saṅgāyana ([devanagari](https://www.tipitaka.org/deva/) [roman](https://www.tipitaka.org/romn/)), you will see there are only a handful of examples of `au` and all of them represent hiatus. (āgantuka_upakkilesa, ariyasāvaka_upāsaka, catutthajjhāna_upamā, eka_usabhagāmī, paritta_udaka, sa_uddesa, sa_udraya, sa_upādāna, sa_upaghāta, sa_upanisa, etc.).

all the dipthongs disappeared in pāli, `au` became `o` (sk d**au**rmanasya / pa d**o**manassa) or `ā` (sk g**au**rava / pa g**ā**rava) or simply a short `u` (sk **au**ddhatya / pa **u**ddhacca). the same happened with `ai`, becoming `e` (sk c**ai**tya / pa c**e**tiya) or `ā` (sk asy**a**i / pa ass**ā** ) or simply `i` (sk c**a**itra / pa c**i**tta). so there's no need to represent something which doesn't exist.

my suggestion would be to make an _IASTprakrit_ for those cases where you need to disambiguate, and perhaps keep _IASTpali_ to its well-established 41 letter set. {a, ā, i, ī, u, ū, e, o, k, kh, g, gh, ṅ, c, ch, j, jh, ñ, ṭ, ṭh, ḍ, ḍḥ, ṇ, t, th, d, dh, n, p, ph, b, bh, m, y, r, l, s, v, h, ḷ, ṃ}
